src/scenic/simulators/newtonian/simulator.py

```diff
@@ -102,10 +107,27 @@ def setup(self):
             )
             self.screen.fill((255, 255, 255))
             x, y, _ = self.objects[0].position
-            self.min_x, self.max_x = min_x - 50, max_x + 50
-            self.min_y, self.max_y = min_y - 50, max_y + 50
+            self.min_x, self.max_x = min_x - 40, max_x + 40
+            self.min_y, self.max_y = min_y - 40, max_y + 40
             self.size_x = self.max_x - self.min_x
             self.size_y = self.max_y - self.min_y
+
+            # Generate a uniform screen scaling (applied to width and height)
+            # that includes all of both dimensions.
+            self.screenScaling = min(WIDTH / self.size_x, HEIGHT / self.size_y)
+
+            # Calculate a screen translation that brings the mean vehicle
+            # position to the center of the screen.
+            self.screenTranslation = (0, 0)
```

Comment:
This line is a bit confusing since the attribute is overwritten below.

Reply:
Hmm, do you have a suggestion to clarify? I wrote it this way since the best way to compute this seemed to be to just see what the Scenic to screen mapping computed with no offset and then center everything.

Reply:
Added a comment to clarify.